addOns/portscan/portscan.gradle.kts

```diff
@@ -10,5 +10,20 @@ zapAddOn {
     manifest {
         author.set("ZAP Dev Team")
         url.set("https://www.zaproxy.org/docs/desktop/addons/port-scan/")
+
+        dependencies {
+            addOns {
+                register("network") {
+                    version.set(">=0.3.0")
+                }
+            }
+        }
     }
 }
+
+dependencies {
+    compileOnly(parent!!.childProjects.get("network")!!)
+
+    testImplementation(parent!!.childProjects.get("network")!!)
+    testImplementation(project(":testutils"))
```

Comment:
It doesn't really matter, but any point adding these without some tests?

Reply:
Copy/paste from other add-on, will remove.

Reply:
Done.